src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -82,6 +82,10 @@ public static void Initialize(Win32PlatformOptions options)
         {
             s_options = options;
 
+#if NET6_0_OR_GREATER
+            ComWrappers.RegisterForMarshalling(Automation.AutomationNodeComWrappers.Instance);
```

Comment:
Note that we should **never** call this from framework code, since there is only one instance per app.